src/CMakeLists.txt

```diff
@@ -5,25 +5,25 @@ set(RandLAPACK_cxx_srcs
     comps/rs.cc
     comps/util.cc
     comps/orth.cc
-    comps/preconditioners.cc
 )
 
 set(RandLAPACK_libs
-    RandLAPACK_RandBLAS
-    RandLAPACK_lapackpp
-    RandBLAS_blaspp
+    RandBLAS
+    lapackpp
+    blaspp
     RandBLAS_Random123
-    RandBLAS_OpenMP
+    OpenMP::OpenMP_CXX
```

Comment:
What effect does removing the ``RandLAPACK_`` and ``RandBLAS_`` prefixes have here? It looks like ``RandBLAS``, ``lapackpp``, and ``blaspp`` will all have semantic interpretations within CMake. That is, CMake will associate the value of the build flag ``-Dlapackpp_DIR=...`` with ``lapackpp``. Is that correct? Maybe this is something for us to discuss on the call.

Reply:
The prefixes disambiguated interface libraries that carried dependency information in the installs. I got rid of those because I think they are not needed. Instead we look for the dependency directly.